examples/07_customers/05_create_customer/query.graphql

```diff
@@ -0,0 +1,16 @@
+mutation customerCreate($input: CustomerCreateInput!) {
```

Comment:
```suggestion
mutation createCustomerAccount($input: CustomerCreateInput!) {
```
Based on [this comment](https://docs.google.com/document/d/1uF-5YBsZJMchnisfOOQOSXimzYudrwcAZykFchrqX5w/edit?disco=AAAAlW_qo5Y) we want the naming to be consistent and action-oriented so for the query and mutation names, the action come first